lib/msf/core/payload_generator.rb

```diff
@@ -252,6 +252,10 @@ def multiple_encode_payload(shellcode)
         @iterations = 1 if iterations < 1
 
         encoder_mod = framework.encoders.create(encoder_opt[0])
+        unless encoder_mod
+          cli_print "#{encoder_opt[0]} not found continuing..."
```

Comment:
When replicating the steps from the original issue:
```
use exploit/windows/smb/psexec
set target 2
set rhost 10.20.36.244
set SERVICE_STUB_ENCODER notfound
run
```

It looks like`cli_print` doesn't output anything to the console for me:

```
msf6 exploit(windows/smb/psexec) > run

[-] Handler failed to bind to 10.12.225.25:4444:-  -
[*] Started reverse TCP handler on 0.0.0.0:4444 
[*] 10.20.36.244:445 - Connecting to the server...
[*] 10.20.36.244:445 - Authenticating to 10.20.36.244:445 as user 'vagrant'...
[!] 10.20.36.244:445 - peer_native_os is only available with SMB1 (current version: SMB2)
[*] 10.20.36.244:445 - Uploading payload... deUFyYUA.exe
[*] 10.20.36.244:445 - Created \deUFyYUA.exe...
[+] 10.20.36.244:445 - Service started successfully...
[*] 10.20.36.244:445 - Deleting \deUFyYUA.exe...
[*] Exploit completed, but no session was created.
```

I put in a breakpoint, and it looks like `cli` is false in this scenario:

![image](https://user-images.githubusercontent.com/60357436/97724441-d1c20e00-1ac4-11eb-9618-30a72450cf14.png)

Reply:
I wonder if that is the case for `msfvenom`? 

It is possible that `cli_print` does not print in a running `msfconsole`, in which case I suggest we consider tackling that separately and possibly enhancing `cli_print` to detect if run from a `shell` or from inside `msfconsole`.

Reply:
Looks like the log and continue is consistent with elsewhere in this file:

https://github.com/rapid7/metasploit-framework/blob/5fbe243662390054fc4dcc4f8250b85068709152/lib/msf/core/payload_generator.rb#L491-L497